app/src/main/java/org/dicekeys/app/encryption/EncryptedDiceKey.kt

```diff
@@ -0,0 +1,20 @@
+package org.dicekeys.app.encryption
+
+import android.util.Base64
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+
+@Serializable
+data class EncryptedDiceKey(
+        @SerialName("key_id")
+        val keyId: String,
+        @SerialName("center_face")
+        val centerFace: String,
+        @SerialName("encrypted_data")
```

Comment:
Does Kotlin allow you to use a constant literal defined somewhere where it can be shared?

Reply:
Yes, its possible.